case_studies/CARD/Bug-Drug Code.R

```diff
@@ -0,0 +1,148 @@
+# config.R
+url <- "https://card.mcmaster.ca/download/0/broadstreet-v3.3.0.tar.bz2"
+destfile <- "broadstreet-v3.3.0.tar.bz2"
+
+# Download the file
+download.file(url, destfile)
+
+#Extract the file
+if (!require("R.utils")) {
+  install.packages("R.utils")
+  library(R.utils)
+}
+
+
+# Extract the tar file
+untar("broadstreet-v3.3.0.tar", exdir = "CARD_data")
+
+
+# Map CARD Short Name
+# Install and Load dplyr
+if (!require("dplyr")) {
+  install.packages("dplyr")
+  library(dplyr)
+} else {
+  library(dplyr)
```

Comment:
since you are using tidyverse, you could use `readr::read_delim` instead of `read.delim`